src/main.cpp

```diff
@@ -8399,12 +8399,6 @@ bool MemorizeMessage(const CTransaction &tx, double dAmount, std::string sRecipi
                         WriteCache("beaconalt",sMessageKey+"."+ToString(nTime),out_publickey,nTime);
                     }
 
-                    // Ensure we have the TXID of the contract in memory
-                    if (!(sMessageType=="project" || sMessageType=="projectmapping" || sMessageType=="beacon" ))
```

Comment:
I'd add sMessageType=="poll" here. Makes it less likely to be braking something.

Reply:
After going through some git history, it appears they are never really used.